tools/public_api_guard/material/form-field.d.ts

```diff
@@ -1,3 +1,5 @@
+export declare const _MAT_HINT: InjectionToken<MatHint>;
```

Comment:
Why is this one underscored, but the others aren't?

Reply:
There should be a JSDoc for the `_MAT_HINT` constant. In short: The MDC-based form-field uses `MatHint` directly, so it's never optional. Hence a lightweight token does not make any sense to have, and to avoid a breaking change when eventually the MDC-based form-field is used, the lightweight token for `MatHint` in non-MDC is private.

Reply:
Would it make sense to just keep it consistent in that case and not use the token? It's not that large to begin with.

Reply:
All of the classes are not really large in source here, but with the generated factory and definition they could easily sum-up. IIRC for hint it's about 230kb. 

I don't think that this should stop us from having a lightweight token in the non-MDC form-field. The form-fields deviate quite a bit anyway already.